operations/meeting-notes.rst

```diff
@@ -45,13 +156,12 @@ treba i placenou. Jeho slovy:
     poměr, tak by to bylo nejjednodušší. Třeba na měsíc. Já jsem odpoveděl, že
     o tom můžu uvažovat, ale minimálně v listopadu a prosinci to dělat nechci.
 
-
 **Úkol**: Javorek se sejde s Bílkem na téma jak poskytuje Pyvce finanční podporu
 např pro PyCon CZ. Poznámky
 [které byly v době zveřejnění zápisu :ref:`již zpracovány <financni-podpora>`]:
 
     * pycon vs pyladies vs pyvo-brno obálky.
-    * ted muze na slacku do #money, vic info neni.
+    * ted muze na slacku do ``#money``, vic info neni.
```

Comment:
When I noticed this change, I decided I'll simplify linking to channels: https://github.com/pyvec/docs.pyvec.org/pull/129

Reply:
ok! perfect! i am going to incorporate that!